qtpy/tests/test_qtcore.py

```diff
@@ -10,9 +14,38 @@ def test_qtmsghandler():
     assert QtCore.qInstallMessageHandler is not None
 
 
-def test_DateTime_toPython():
+def test_qdatetime_toPython():
     """Test QDateTime.toPython"""
+    q_date = QtCore.QDateTime.currentDateTime()
     assert QtCore.QDateTime.toPython is not None
+    py_date = q_date.toPython()
+    assert isinstance(py_date, datetime)
+
+
+@pytest.mark.skipif(sys.platform.startswith('linux') and os.environ.get('USE_CONDA', 'No') == 'No',
```

Comment:
Given that checking for conda packages is used a lot in this PR, I suggest to create a proper function for it in `qtpy/tests/utils.py` like this

```python
def use_conda():
    return os.environ.get('USE_CONDA') == 'Yes'
```

That would help to better understand these skips and also and to use it in the future in other tests as well.